openassessment/xblock/static/sass/oa/elements/_staff.scss

```diff
@@ -17,34 +17,31 @@
     border-top: ($baseline-v/4) solid $color-decorative-staff;
     padding: $baseline-v ($baseline-h/2);
     background: $staff-bg;
+    display: none;
```

Comment:
Note to reviewers: for collapsible items, the display is initially set to none in the CSS (since they are initially collapsed), and then JQuery slideUp/slideDown changes the display as necessary.